.all-contributorsrc

```diff
@@ -252,6 +252,15 @@
       "contributions": [
         "content"
       ]
+    },
+    {
```

Comment:
Hope you can remove this, as the bot all-contributors does this automatically when called!

Reply:
I 've removed changes in all-contributors and added a new quote! :)

Reply:
Thanks for fixing it!! 👍